hackathon_site/dashboard/frontend/src/pages/AdminDashboard/AdminDashboard.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import AdminDashboard from "pages/AdminDashboard/AdminDashboard";
+
+import { render, waitFor } from "testing/utils";
+
+describe("Admin Dashboard Page", () => {
```

Comment:
Can you make a test that checks for Pending order Cards similar to how `Orders.test.tsx` checks for orders. Refer to that file to see how they implement the store
```typescript
test("Display correct order cards", () => {
        const { getByTestId } = render(<Orders />, { store });
        mockPendingOrders.forEach((order) => {
            const orderStatus = order.status;

            if (orderStatus === "Submitted" || orderStatus === "Ready for Pickup") {
                const orderItem = getByTestId(`order-item-${order.id}`);
                const orderDetails = within(orderItem);
                const date = new Date(order.created_at);
                const month = date.toLocaleString("default", { month: "short" });
                const day = date.getDate();
                const hoursAndMinutes = date.getHours() + ":" + date.getMinutes();

                expect(orderDetails.getByText(order.id)).toBeInTheDocument();
                expect(orderDetails.getByText(order.items.length)).toBeInTheDocument();
                expect(orderDetails.getByText(order.team_code)).toBeInTheDocument();
                expect(
                    orderDetails.getByText(`${month} ${day}, ${hoursAndMinutes}`)
                ).toBeInTheDocument();
            }
```

Reply:
I added the test similar to orders.test.tsx. This test uses getByTestId and the data-testid attribute so I implemented it into the admindashboard:
<img width="433" alt="image" src="https://github.com/ieeeuoft/hackathon-template/assets/54194957/27765c3a-e961-49a9-b342-a3fc623aac6b">

But it wouldn't work when I had `id={order.team_id}`. I had to change it to `id={order.id}` for the data-testid to show up. 

What I mean is that when I had the following:

```
id={order.team_id}
status={order.status}
data-testid={`order-item-${order.id}`}
```
The HTML would display `data-testid="order-item-2"` which was the order.team_id rather than the order.id

When I changed it to
```
id={order.id}
status={order.status}
data-testid={`order-item-${order.id}`}
```
Then, it worked properly and the HTML displayed `data-testid="order-item-3"` which is the correct number (order.id). I'm not sure why this happens. Do you know what the issue could be?

Reply:
In general, it is not recommended to render components with the same ID. The initial issue might have been caused by rendering multiple components with the same ID, especially considering that teams can have multiple order cards. The second approach would ensure its a unique ID.

Reply:
Hmm, interesting okay. Yeah I think this way of displaying the ID as the order.id rather than the team_id works better anyways. We already display the team_code so we should display the order id not the team_id.